src/main/java/gregtech/api/recipes/RecipeMap.java

```diff
@@ -697,9 +768,9 @@ private Recipe recurseIngredientTreeFindRecipe(@NotNull List<List<AbstractMapIng
      * @return a recipe
      */
     @Nullable
-    private Recipe recurseIngredientTreeFindRecipe(@NotNull List<List<AbstractMapIngredient>> ingredients,
-                                                   @NotNull Branch branchMap, @NotNull Predicate<Recipe> canHandle,
-                                                   int index, int count, long skip) {
+    public Recipe recurseIngredientTreeFindRecipe(@NotNull List<List<AbstractMapIngredient>> ingredients,
```

Comment:
`protected` probably? Since the iterator class can access it with package privacy, and there's no need to expose this publicly

Reply:
fixed